src/ui/public/kbn_top_nav/kbn_top_nav_controller.js

```diff
@@ -22,6 +23,10 @@ export default function ($compile) {
       this.addItems(opts);
     }
 
+    shouldBeVisible() {
```

Comment:
I think `isVisible` would be better here because it's more conventional and shorter (and I can't discern a semantic difference between `is` and `shouldBe` in the context of flags).